src/cmd/update.js

```diff
@@ -55,12 +61,33 @@ module.exports = class UpdateCommand {
 			return;
 		}
 
+		if (!devInfo.inDfuMode && (!devInfo.version || semver.lt(devInfo.version, '2.0.0'))) {
+			// The support for control requests is somewhat inconsistent in pre-LTS versions of Device OS
+			console.log(chalk.red('!'), 'The version of Device OS that is running on the device is too old.');
```

Comment:
It's not necessary to give this information to the user. The next line (please switch to DFU) is sufficient.